tailwind.config.js

```diff
@@ -34,10 +35,11 @@ module.exports = {
       white: {
         DEFAULT: '#FFFFFF',
       },
-      grey: {
+      gray: {
         black: '#13191B',
         darker: '#293139',
         dark: '#333D46',
+        dark_mid: '#42525C',
```

Comment:
replace all this names by numbers to have a better meaning. Otherwise it will lead to questions like "Which is darker: `dark` or `dark_mid`"? Tailwind itself knows about this problem and uses number by default: https://tailwindcss.com/docs/customizing-colors#color-object-syntax